tin/apps/courses/views.py

```diff
@@ -100,7 +112,8 @@ def edit_view(request, course_id):
             course = form.save()
             return redirect("courses:show", course.id)
     else:
-        form = CourseForm(instance=course)
+        initial = {"permission": "r" if not course.archived else course.permission}
```

Comment:
Do you need this? If it's a database field from a ModelForm, it should be pre-populated, right?